core/schema/file.graphqls

```diff
@@ -15,7 +15,7 @@ type File {
   contents: String!
 
   "Retrieves a secret referencing the contents of this file."
-  secret: Secret! @deprecated(reason: "insecure, leaves secret in cache")
+  secret: Secret! @deprecated(reason: "insecure, leaves secret in cache. Superseded by setSecret")
```

Comment:
Didn't get a chance to review. References to new fields in a deprecation message should be wrapped in backticks. That allows the SDKs to rename the new field according to language conventions. This is undocumented for now, but will be included in the forthcoming API coding style issue.

```suggestion
  secret: Secret! @deprecated(reason: "insecure, leaves secret in cache. Superseded by `setSecret`")
```

Reply:
done in https://github.com/dagger/dagger/pull/4885